### PR TITLE
Remove duplicate code

### PR DESCRIPTION
**Pull request by GermanAizek**
@andrepuschmann 
Cleaning dups code, args_default body how structure fields are specified, fixed spelling error in comment

### CODE REVIEW

File: lib/examples/pssch_ue.c [hunk 1]
```diff
@@ -80,7 +79,6 @@ void args_default(prog_args_t* args)
 }
 
 static srsran_pscch_t pscch = {}; // Defined global for plotting thread
-static srsran_pssch_t pssch = {};
```
Comment (andrepuschmann) [inline lib/examples/pssch_ue.c:83]: this is definitely used, how would it even compile without?

File: test/phy/nr_phy_test.cc [hunk 2]
```diff
@@ -206,11 +206,6 @@ test_bench::args_t::args_t(int argc, char** argv)
     gnb_stack.pdsch.rb_start  = 0;
   }
 
-  if (gnb_stack.pusch.rb_length == 0) {
```
Comment (andrepuschmann) [inline test/phy/nr_phy_test.cc:209]: and this?

**Review by andrepuschmann — COMMENTED** [review]
thanks for the PR. i have a few doubts regarding the changes, please see below

**Comment by GermanAizek** [comment]
@andrepuschmann, fixed commit after code review here https://github.com/srsran/srsRAN/pull/1052/commits/706597a3174421ef715781e69a8d5e0080a9029d

**Comment by andrepuschmann** [comment]
@GermanAizek Ok, so in the end the other changes weren't needed. Fair enough. Can you sign the CLA as well please?

**Comment by GermanAizek** [comment]
@andrepuschmann I have already signed it in this PR https://github.com/srsran/srsRAN/pull/1051

**Comment by andrepuschmann** [comment]
Included in release. Thanks again

**Pull request closed without merging**